crates/rmf_site_editor/src/interaction/select_impl/place_object.rs

```diff
@@ -62,14 +61,14 @@ pub struct ObjectPlacement<'w, 's> {
 }
 
 impl<'w, 's> ObjectPlacement<'w, 's> {
-    pub fn place_object_2d(&mut self, object: ModelInstance<Entity>) {
-        let Some(level) = self.current_level.0 else {
-            warn!("Unble to create [object:?] outside a level");
-            return;
-        };
```

Comment:
This introduced a warning for `current_level` not being used anymore in the struct

Reply:
Fixed: 39947a9ad2605a4052071253354a4f9c126e1072